src/System.Memory/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Decimal.E.cs

```diff
@@ -0,0 +1,106 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Buffers.Text
+{
+    public static partial class Utf8Formatter
+    {
+        private static bool TryFormatDecimalE(ref NumberBuffer number, Span<byte> buffer, out int bytesWritten, byte precision, byte exponentSymbol)
+        {
+            const int NumExponentDigits = 3;
```

Comment:
move to Utf8Constants?

Reply:
The "3" is specific to the "E" format. The "G" format specifies "2". And I haven't looked up what the rules for Double and Single are. Since the BCL didn't bother to be consistent here, I think it's best to keep these as local constants.